packages/js/data/src/crud/utils.ts

```diff
@@ -168,3 +215,46 @@ export const cleanQuery = (
 
 	return cleaned;
 };
+
+/**
+ * Get the identifier for a request provided its arguments.
+ *
+ * @param  name Name of action or selector.
+ * @param  args Arguments for the request.
+ * @return Key to identify the request.
+ */
+export const getRequestIdentifier = ( name: string, ...args: unknown[] ) => {
```

Comment:
I'm avoiding using `resourceName` here since I don't think that's accurate and is ambiguous with the actual resource name (e.g., `Products`).

But I could see this also being `getRequestKey` or open to other suggestions.